src/main/java/com/aetherteam/aether_genesis/data/generators/GenesisLanguageData.java

```diff
@@ -35,17 +36,39 @@ protected void addTranslations() {
         this.addItem(GenesisItems.BLUE_SWET_JELLY, "Blue Swet Jelly");
         this.addItem(GenesisItems.GOLDEN_SWET_JELLY, "Golden Swet Jelly");
         this.addItem(GenesisItems.DARK_SWET_JELLY, "Dark Swet Jelly");
+        this.addItem(GenesisItems.ICESTONE_POPROCKS, "Icestone Poprocks");
+        this.addItem(GenesisItems.COCOATRICE, "Cocoatrice");
+        this.addItem(GenesisItems.WARPED_CHOCOLATES, "Warped Chocolate");
+        this.addItem(GenesisItems.BLUEBERRY_LOLLIPOP, "Blueberry Lollipop");
+        this.addItem(GenesisItems.ORANGE_LOLLIPOP, "Orange Lollipop");
+        this.addItem(GenesisItems.PINK_SWET_LOLLIPOP, "Stomper Pop");
+        this.addItem(GenesisItems.ORANGE, "Orange");
+        this.addItem(GenesisItems.WYNDBERRY, "Wyndberry");
+        this.addItem(GenesisItems.JELLY_PUMPKIN, "Jelly Pumpkin");
+        this.addItem(GenesisItems.CANDY_CORN, "Candy Corn");
+        this.addItem(GenesisItems.RAINBOW_STRAWBERRY, "Rainbow Strawberry");
+
+        this.addItem(GenesisItems.SKYROOT_BOWL, "Skyroot Bowl");
+        this.addItem(GenesisItems.CORNSTARCH_BOWL, "Cornstarch Bowl");
+        this.addItem(GenesisItems.CRYSTAL_EXP_BOTTLE, "Crystal Exp Bottle");
+        this.addItem(GenesisItems.BONE_RING, "Bone Ring");
+
         this.addItem(GenesisItems.DARK_SWET_SPAWN_EGG, "Dark Swet Spawn Egg");
+        this.addItem(GenesisItems.TEMPEST_SPAWN_EGG, "Tempest Spawn Egg");
         this.addItem(GenesisItems.BLUE_PARACHUTE, "Blue Parachute");
         this.addItem(GenesisItems.GREEN_PARACHUTE, "Green Parachute");
         this.addItem(GenesisItems.PURPLE_PARACHUTE, "Purple Parachute");
 
         this.addEntityType(GenesisEntityTypes.DARK_SWET, "Dark Swet");
+        this.addEntityType(GenesisEntityTypes.TEMPEST, "Tempest");
 
         this.addCommonConfig("gameplay", "gold_aercloud_ability", "Changes Gold Aercloud and respective parachute behavior to launch entities downwards");
 
         this.addPackDescription("mod", "Genesis Resources");
 
+        this.addBlock(GenesisBlocks.AETHER_CRAFTING_TABLE, "Skyroot WorkBench");
```

Comment:
This is a nitpick, but the addBlock lines should be organized between the addItem ones and the addEntityType ones

Reply:
no nitpick, i need to do

Reply:
Skyroot Crafting Table should also be named as such in its localized name and field name

Reply:
okay, will fix it